### ISSUE TITLE
Recolour links on data service

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#1197]
**Please complete the following information**
Data repository URL: All
Data or metadata issue: Data service

Link: https://sdgdata.gov.uk/sdg-data/disaggregations.html

Reuse open-sdg link colour (also GDS suggested) for styling links on data service

**Additional context**
Current colours don't meet constrast guidelines in some places

**Comment by phillipgardiner** [comment]
@brockfanning 
link-colour = #1d70b8
link-hover-colour = #003078
link-visited-colour = #4c2c92

**Comment by brockfanning** [comment]
@phillipgardiner I just wanted to confirm that `#1d70b8` is correct, because that's what is already in place for the link color.

**Comment by phillipgardiner** [comment]
@brockfanning Yes #1d70b8 is correct: https://design-system.service.gov.uk/styles/colour/. Had to check back to the original report to remind myself about this. Essentially the blue link colour fails contrast rules on the current row grey background used.  If the grey background is changed as per https://github.com/open-sdg/open-sdg/issues/1198 this will then pass. So essentially continue to use #1d70b8 but change the grey used on rows. 

I believe this ticket is also about getting the link-hover-colour and link-visited-colour correct too. 

Hope that makes sense?

**Comment by brockfanning** [comment]
@phillipgardiner Makes sense. All three of these data-service-styling tickets are part of the same PR (https://github.com/open-sdg/sdg-build/pull/262) so it should be possible test/review once that is on a feature branch.
